RAGworkshop/README.md

```diff
@@ -0,0 +1,440 @@
+4. Connection details - Instructions on how to connect to the Kafka cluster that hosts this feed. 
+
+The feed we are interested in is named “Youtube Gaming Subtitles” and can be found in the “Web and Tech” category. Under connection details you should see two sets of connection details:
+
+1. Operational connection details - these can be used to investigate the data for operational purposes  
+   
+```
+`bootstrap.servers=pkc-l6wr6.europe-west2.gcp.confluent.cloud:9092`  
+`security.protocol=SASL_SSL`  
+`sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username=XXXXXX password='XXXXXXX';`  
+`sasl.mechanism=PLAIN`
+```
+
+2. Analytical connection details - these are to be used with analytical tooling for more complex, SQL based queries.   
+   
+```
+`SET SESSION streambased_connection = '{`  
+  `"bootstrap.servers": "pkc-l6wr6.europe-west2.gcp.confluent.cloud:9092",`  
+  `"security.protocol": "SASL_SSL",`  
+  `"sasl.jaas.config": "org.apache.kafka.common.security.plain.PlainLoginModule required username=''XXXXXXXXXXXX'' password=''XXXXXXXXX'';",`  
+  `"sasl.mechanism": "PLAIN",`  
+  `"schema.registry.url":"https://psrc-571d82.europe-west2.gcp.confluent.cloud",`  
+  `"basic.auth.credentials.source":"USER_INFO",`  
+  `"basic.auth.user.info":"XXXXXX"`  
+`}';`
+```
+
+Copy both of these connection details for use later.
+
+**Step 2: Explore the data using analytical tools**
+
+Whilst it is possible to achieve exploratory tasks with the above tools they are very clunky and every task becomes a multi-tool, multi-stage process. Realtime data doesn’t have to be like this however, let’s perform the same tasks using analytical tools.
+
+Note: This section uses Streambased A.S.K., a free, hosted analytical tool for Kafka based on the open source Trino project. 
+
+Streambased A.S.K. offers a JDBC/ODBC/SQL Alchemy connection and so can be integrate with most analytical tools. We love the simplicity of [SquirrelSQL](https://squirrel-sql.sourceforge.io/) but feel free to follow the guides below to connect the tool of your choice.
+
+[https://www.streambased.io/tutorial-guides](https://www.streambased.io/tutorial-guides)
+
+Remember the analytical connection details we collected from RDx? We can apply these with a single command in our SQL tool to repoint A.S.K. at our Kafka data.
+
+```
+`SET SESSION streambased_connection = '{`  
+  `"bootstrap.servers": "pkc-l6wr6.europe-west2.gcp.confluent.cloud:9092",`  
+  `"security.protocol": "SASL_SSL",`  
+  `"sasl.jaas.config": "org.apache.kafka.common.security.plain.PlainLoginModule required username=''XXXXXXXXXXXX'' password=''XXXXXXXXX'';",`  
+  `"sasl.mechanism": "PLAIN",`  
+  `"schema.registry.url":"https://psrc-571d82.europe-west2.gcp.confluent.cloud",`  
+  `"basic.auth.credentials.source":"USER_INFO",`  
+  `"basic.auth.user.info":"XXXXXX"`  
+`}';`
+```
+
+1. List topics \- for an analytical view of Kafka data we apply the following mappings:  
```

Comment:
this feels like it needs the console tools explanation, rewrite it with less of a focus on console vs analytical